cev_eris.dme

```diff
@@ -2334,7 +2338,6 @@
 #include "code\modules\projectiles\guns\projectile\automatic\lmg.dm"
 #include "code\modules\projectiles\guns\projectile\automatic\maxim.dm"
 #include "code\modules\projectiles\guns\projectile\automatic\molly.dm"
-#include "code\modules\projectiles\guns\projectile\automatic\slaught_o_matic.dm"

```

Comment:
What

Reply:
Probably got picked off while I was swapping branches. Easy to fix.